### PR TITLE
Show and edit specific task within specific subject

### PR DESCRIPTION
**Pull request by duypn-2245**

### CODE REVIEW

**Comment by duypn-2245** [comment]
image
image
image
image
image

**Comment by duypn-2245** [comment]
https://edu-redmine.sun-asterisk.vn/issues/25374

**Comment by duypn-2245** [comment]
ready

**Review by tham-1781 — APPROVED** [review]

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    flash[:warning] = t "subjects.load_subject.not_found"
+    redirect_to root_path
+  end
+
+  def load_task
+    @task = Task.find_by id: params[:id]
+    return if @task
+
+    flash[:warning] = t "tasks.load_task.not_found"
```
Comment (vanvtt-0952) [inline app/controllers/tasks_controller.rb:37]: (tat) flash rồi sao chú

File: app/views/tasks/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<%= provide :title, t(".title") %>
+<h2><%= t ".title" %></h2>
+<div class="col-md-12">
+  <%= render "task_form" %>
+  <hr>
+  <%= link_to t(".back_to_subject"), @task.subject, class: "btn btn-primary" %>
+</div>
+
+
```
Comment (vanvtt-0952) [inline app/views/tasks/edit.html.erb]: dư line trống nè

**Comment by duypn-2245** [comment]
ready

**Pull request merged**